repo/linux/mptcp

```diff
@@ -1,6 +1,6 @@
 url: https://github.com/multipath-tcp/mptcp_net-next.git
-integration_testing_branches: export
-branch_allowlist: export
+integration_testing_branches: export|export-net
```

Comment:
Hi, the rule of integration_testing_branches is different, it's like the below one:
```
integration_testing_branches:
- export
- export-net
```

could you mind resending the patch?

Reply:
No problem, I'll update it.